test/webgpu_basic_rendering.cpp

```diff
@@ -54,7 +54,11 @@ void GetAdapter(void (*callback)(wgpu::Adapter)) {
 }
 
 void GetDevice(void (*callback)(wgpu::Device)) {
-    adapter.RequestDevice(nullptr, [](WGPURequestDeviceStatus status, WGPUDevice cDevice, const char* message, void* userdata) {
+    wgpu::DeviceDescriptor desc{};
+    desc.uncapturedErrorCallbackInfo = {nullptr, [](WGPUErrorType type, char const* message, void*) {
```

Comment:
Let's wait until @lokokung's 🪄 then ;)

Reply:
I believe https://dawn-review.googlesource.com/c/dawn/+/192741 should help

Reply:
@lokokung @kainino0x Out of curiosity, what's the current status?

Reply:
For Emscripten changes, we recently pulled a fork of Emscripten into Dawn and are planning to make all changes there for a while and eventually upstream all the changes once we are closer to a stable header. As to when/where to land this change though, I'm not entirely sure and will let @kainino0x comment.

Reply:
I think we can add it to upstream emscripten, if we want, as soon as we are pretty sure the API shape for DeviceDescriptor is stable in Dawn (in both C and C++). But I'm not sure it's really worth it to do incrementally now instead of just waiting until we're doing other changes. In particular, adding fields to the descriptor can break dependent code, if they're not zero-initializing the whole struct. That should be relatively rare and they can just add zero-init but it does add some extra friction to do it now instead of later (IMO).

Reply:
@lokokung This seems related to https://github.com/beaufortfrancois/webgpu-cross-platform-app/pull/8#issuecomment-2296481798 as well.

Reply:
Similarly to https://github.com/emscripten-core/emscripten/issues/21072#issuecomment-2324671652, I believe we can close this issue as next time we upstream changes in Emscripten, uncapturedErrorCallbackInfo will be implemented properly according to https://github.com/emscripten-core/emscripten/issues/21552#issuecomment-2319481925
@kainino0x What do you think?

Reply:
Sounds good to me.